### DISCUSSION TITLE
考虑更新一下文档

### DISCUSSION THREAD

**Discussion by cath-gh** [ppoffice/hexo-theme-icarus#1119] [General]
[帮助文档](https://ppoffice.github.io/hexo-theme-icarus/)上的内容比较早了，和现在版本号差了不少，是否考虑更新一下文档

**Comment by ppoffice** [comment]
最近一次更新也才一个月前呀？
https://github.com/ppoffice/hexo-theme-icarus/commits/site

**Reply by cath-gh** [reply]
文章底部不显示更新日期，我以为还是原来的呢，不好意思
